.github/ISSUE_TEMPLATE/sprint_planning_retro.md

```diff
@@ -20,6 +20,10 @@ For your team sprint planning copy this template into a comment below for each t
 
 **Support hero:** ___
 
+## Changelog
+Things @joethreepwood should add to the changelog. PR links appreciated. 
```

Comment:
@joethreepwood I think it'd be helpful if you were more descriptive about what should be included in changelog here. How big does a feature or fix need to be?

Reply:
Hm I still don't think this answers the question of what counts as a feature or improvement that clears the bar for the changelog?